packages/plugin-ext/src/plugin/node/debug/debug.spec.ts

```diff
@@ -0,0 +1,93 @@
+/********************************************************************************
+ * Copyright (C) 2022 STMicroelectronics and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { DebugSession } from '@theia/plugin';
+import * as chai from 'chai';
+import { ProxyIdentifier, RPCProtocol } from '../../../common/rpc-protocol';
+
+import { DebugExtImpl } from './debug';
+
+const expect = chai.expect;
+
+describe('Debug source URI:', () => {
```

Comment:
minor: in general we have a parent `describe` that describes the file, then we have individual tests:

```suggestion
describe('Debug API', () => {

    describe('#asDebugSourceUri', () => {
```